getStarted/typography/typography.css

```diff
@@ -0,0 +1,48 @@
+/* font-sizes */
+.x-large-text {
+  font-size: var(--font-size-1);
+}
+
+.large-text {
+  font-size: var(--font-size-2);
+}
+
+.medium-text {
+  font-size: var(--font-size-3);
+}
+
+.small-text {
+  font-size: var(--font-size-4);
+}
+
+.x-small-text {
+  font-size: var(--font-size-5);
+}
```

Comment:
The sizes are not getting applied. Try importing the stylesheet where these variables are defined.

Reply:
Thanks for pointing fixing it now